### ISSUE TITLE
Conjure injects `package.loaded.fennel` but does not include `fennel.traceback`

### ISSUE THREAD

**Issue by rktjmp** [Olical/conjure#237]
See demo

https://github.com/rktjmp/conjure-fennel-traceback

And related issue, the issue's an open debug log so, kind of a mess, probably just the first and last post are interesting. https://github.com/rktjmp/hotpot.nvim/issues/7 

When running conjure, it loads (I assume meta-table-proxies) Fennel into `package.loaded["fennel"]`, but it does not proxy calls to `package.loaded["fennel"].traceback`.  

This causes the following Fennel `open-with` check to fail:

` return close_handlers_0_(xpcall(_0_, (package.loaded.fennel or debug).traceback))`

Because `package.loaded.fennel` is points to an aniseed table, but the table does not include a `traceback` function.

Would it be possible to proxy traceback (unsure of side effects if whole fennel lib not loaded), or not load into `package.loaded.fennel`? (Maybe `package.loaded["aniseed.fennel"]`?)

**Comment by Olical** [comment]
So it's also only a problem if you have `use_metadata` enabled I think?

**Comment by Olical** [comment]
Pushed a fix to the Conjure `develop` branch and moved this issue back to the repo, you were right all along, thank you for persisting with my ignorance 😅

**Comment by rktjmp** [comment]
> So it's also only a problem if you have use_metadata enabled I think?

I am just using the default setup, but yeah looks that way.

Fix looks good to me, test passes. :+1:

**Comment by Olical** [comment]
Excellent! It also won't override an existing `package.loaded.fennel` now, if there's something already there it'll leave it intact. This bug arose from some major refactoring I did in the Aniseed repo a while back, I moved things around and only loaded the fennel compiler when I absolutely needed to. This meant the fennel module moved location changed, breaking the code in the Conjure repo.

I broke my own code with my own breaking change in a hidden undocumented part of the source 🙃 I'll get some releases out now.
